src/index.tsx

```diff
@@ -197,6 +197,10 @@ export interface ReactTagsWrapperProps {
    * Whether to display a button to clear all the tags.
    */
   clearAll?: boolean;
+  /**
+   * accessibility la
+   */
+  accessibilityLabelForClearAll?: string;
```

Comment:
The prop `accessibilityLabelForClearAll` is descriptive, but a shorter name like `clearAllLabel` could make the code more concise and still clear. This is subjective, but shorter names tend to improve readability.

Reply:
my bad , I thought jsdoc is added looking at auto-complete, will add it

Reply:
thanks for the suggestion. My concern is `clearAllLabel` might give the impression to the user that this value is used inside the button as text. What do you think?